components/api/src/main/java/com/hotels/styx/api/FullHttpResponse.java

```diff
@@ -43,7 +43,26 @@
 import static java.util.stream.Collectors.toList;
 
 /**
- * HTTP response with a fully aggregated/decoded body.
+ * An immutable HTTP response object including full body content.
+ * <p>
+ * A {@link FullHttpResponse} is useful for response messages with a
```

Comment:
If you agree with the comment for `FullHttpRequest`, the same applies here.

Reply:
Okay I have replaced "response messages" with just "responses".